packages/inspection-report/src/components/DamageReport/DamageReport.js

```diff
@@ -51,16 +54,23 @@ const styles = StyleSheet.create({
 function DamageReport() {
   const { t } = useTranslation();
   const [tabIndex, setTabIndex] = useState(0);
+  const [isPopVisible, setIsPopVisible] = useState(false);
```

Comment:
rename to `isPopUpVisible, setIsPopUpVisible`

Reply:
Done!